src/server/common/model/section/section-model/section-model-updated.test.js

```diff
@@ -67,10 +86,12 @@ describe('SectionModel.validate', () => {
   })
 
   // Reason: We have not finalised how exit pages will behave
-  it.skip('should return ... invalid ? ... if the section hits an exit condition before its complete', () => {
+  it('should return invalid if the section hits an exit condition before its complete', () => {
     const origin = Origin.fromState(exitState)
+    const { isValid, firstInvalidPage } = origin.validate()
 
-    expect(origin.validate().isValid).toBe(true)
+    expect(isValid).toBe(false)
+    expect(firstInvalidPage).toBeInstanceOf(OnOffFarmPage)
```

Comment:
I guess we need to try and find a way to test the section model without references to specific pages like "OnOffFarmPage" at some point

Reply:
Yes, I think so.  We don't _yet_ have a great answer to this, but I'm hoping to wait until we've got a bit more experience with this current implementation.